### PR TITLE
fix: [SRM-10795]: Deployment card and K8s card UI

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
##### Summary:

Summary - Making UI font and layout changes to Deployment card and K8s card:

Infrastructure change details panel implementation is not matching with the mock-ups. Update the font size, color, spacing, and verbiage as in mock-ups. Please refer to the Figma mocks.

Deployment change details panel mplementation is not matching with the mock-ups. Update the font size, color, spacing, and verbiage as in mock-ups. Please refer to the Figma mocks.

The verification status on the Deployment change details card under "Deployment Health" needs to be enhanced. To make the most of the space, I arranged the status and legend in one row. Please refer to the attached image.

##### Jira Links:

https://harness.atlassian.net/browse/SRM-10795
https://harness.atlassian.net/browse/SRM-10796

##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
image

**Comment by vasuki-kalluri** [comment]
retrigger eslint

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Comment by vasuki-kalluri** [comment]
retrigger jest

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx [hunk 1]
```diff
@@ -6,47 +6,61 @@
...
-export default function VerificationStatusCard({ status }: { status: VerifyStepSummary['verificationStatus'] }) {
+export default function VerificationStatusCard({
+  status
+}: {
+  status: VerifyStepSummary['verificationStatus'] | PipelineExecutionSummary['status']
+}) {
   let statusMessage: string | undefined = undefined
   let color: Color | undefined = undefined
   let backgroundColor: Color | undefined = undefined
+  let icon: any = undefined
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx:24]: types should be IconName

File: src/modules/85-cv/components/ServiceDependenciesLegend/ServiceDependenciesLegend.tsx [hunk 2]
```diff
@@ -26,6 +26,9 @@ const ServiceDependenciesLegend: React.FC<ServiceDependenciesLegendProps> = ({ h
   return (
     <Container background={Color.PRIMARY_1} width="fit-content" className={css.container} margin={margin}>
       <Layout.Horizontal flex spacing="medium" height={30} padding={{ left: 'medium', right: 'medium' }}>
+        <Text font={{ size: 'small', weight: 'semi-bold' }} color={Color.BLACK_100}>
+          Service Health:
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ServiceDependenciesLegend/ServiceDependenciesLegend.tsx:30]: move to strings

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/ChangeEventServiceHealth/ChangeEventServiceHealth.module.scss [hunk 3]
```diff
@@ -8,16 +8,31 @@
...
     font-weight: bolder !important;
-    font-size: var(--font-size-medium) !important;
+    font-size: var(--font-size-normal) !important;
     margin-bottom: var(--spacing-small) !important;
     color: var(--black) !important;
   }
 
   .healthTrend {
-    font-weight: bolder !important;
+    //font-weight: bolder !important;
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/ChangeEventServiceHealth/ChangeEventServiceHealth.module.scss:17]: remove comment

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/ChangeDetails/ChangeDetails.tsx [hunk 4]
```diff
@@ -15,30 +15,43 @@ import { useStrings } from 'framework/strings'
 import { ChangeSourceTypes } from '@cv/pages/ChangeSource/ChangeSourceDrawer/ChangeSourceDrawer.constants'
 import { EXECUTED_BY } from '@cv/constants'
 import type { ChangeEventDTO } from 'services/cv'
-import { createDetailsTitle, getOnClickOptions, statusToColorMapping } from './ChangeDetails.utils'
+import { getOnClickOptions, statusToColorMapping } from './ChangeDetails.utils'
 import type { ChangeDetailsDataInterface } from '../../ChangeEventCard.types'
 import StatusChip from './components/StatusChip/StatusChip'
 import css from './ChangeDetails.module.scss'
 
+const getDetailsLabelStringId = (key: string): string => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/ChangeDetails/ChangeDetails.tsx:23]: move to utils file

**Comment by vasuki-kalluri** [comment]
Summary - Making UI font and layout changes to Deployment card and K8s card:

Infrastructure change details panel implementation is not matching with the mock-ups. Update the font size, color, spacing, and verbiage as in mock-ups.  Please refer to the Figma mocks. 

Deployment change details panel mplementation is not matching with the mock-ups. Update the font size, color, spacing, and verbiage as in mock-ups.  Please refer to the Figma mocks. 

The verification status on the Deployment change details card under "Deployment Health" needs to be enhanced. To make the most of the space, I arranged the status and legend in one row. Please refer to the attached image.

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/ChangeEventServiceHealth/ChangeEventServiceHealth.module.scss (hunk 3 above)
Comment (vasuki-kalluri) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/ChangeEventServiceHealth/ChangeEventServiceHealth.module.scss:17]: Done

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx (hunk 1 above)
Comment (vasuki-kalluri) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx:24]: Done

File: src/modules/85-cv/components/ServiceDependenciesLegend/ServiceDependenciesLegend.tsx (hunk 2 above)
Comment (vasuki-kalluri) [reply src/modules/85-cv/components/ServiceDependenciesLegend/ServiceDependenciesLegend.tsx:30]: Done

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx [hunk 5]
```diff
@@ -6,47 +6,61 @@
...
+  status: VerifyStepSummary['verificationStatus'] | PipelineExecutionSummary['status']
+}) {
   let statusMessage: string | undefined = undefined
   let color: Color | undefined = undefined
   let backgroundColor: Color | undefined = undefined
+  let icon = ''
   const { getString } = useStrings()
   switch (status) {
     case 'IN_PROGRESS':
+    case 'Running':
```
Comment (harshilbits) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx:28]: Please make an enum of different status or reuse the existing enum

File: src/modules/85-cv/utils/CommonUtils.ts [hunk 6]
```diff
@@ -248,3 +248,18 @@ export const getTags = (tags?: CVNGLogTag[]) => {
 
   return JSON.stringify(result)
 }
+
+export const getDetailsLabelStringId = (key: string): string => {
+  switch (key) {
+    case 'artifactType':
+      return 'Artifact Type'
```
Comment (harshilbits) [inline src/modules/85-cv/utils/CommonUtils.ts:255]: These should be in strings yaml

Comment (vasuki-kalluri) [reply src/modules/85-cv/utils/CommonUtils.ts:255]: Done

**Review by deepesh-ui — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Pull request merged**